react/utils/assets.ts

```diff
@@ -130,8 +130,8 @@ export function getExtensionImplementation<P={}, S={}>(extensions: Extensions, n
   return extension && extension.component ? getImplementation<P, S>(extension.component) : null
 }
 
-export function fetchAssets(account: string, assets: string[]) {
-  const absoluteAssets = assets.map(url => getAbsoluteURL(account, url))
+export function fetchAssets(runtime: RenderRuntime, assets: string[]) {
+  const absoluteAssets = assets.map(url => getAbsoluteURL(runtime.account, url, runtime.workspace, runtime.production))
```

Comment:
isn't it clearer destructuring before using the variables ?
```suggestion
  const { account, workspace, production } = runtime
```